.travis.yml

```diff
@@ -39,9 +39,13 @@ matrix:
     - php: 7.4
       env:
         - DEPS=latest
+    - php: 'nightly'
+      env:
+        - DEPS=latest
+        - COMPOSER_ARGS="--no-interaction --ignore-platform-reqs"
```

Comment:
If you added `^8.0`, then doing `--ignore-platform-reqs` here shouldn't be required

Reply:
It's required temporarily as other dev deps like `php-http/psr7-integration-tests` and `laminas/laminas-zendframework-bridge` don't get installed without it as their deps also need to be updated to allow installing on PHP 8.

https://travis-ci.com/github/laminas/laminas-diactoros/jobs/372193687#L225